llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1012,9 +1016,42 @@ Error MCCASBuilder::splitDebugInfoCUs() {
   return finalizeSection();
 }
 
+Error MCCASBuilder::createLineSection(const MCSection *DebugLineSection) {
+  if (!DebugLineSection)
+    return Error::success();
+
+  Expected<SmallVector<char, 0>> DebugLineData =
+      mergeMCFragmentContents(DebugLineSection->getFragmentList(), true);
+  if (!DebugLineData)
+    return DebugLineData.takeError();
+
+  startSection(DebugLineSection);
+
+  StringRef DebugLineStrRef =
+      StringRef(DebugLineData->data(), DebugLineData->size());
```

Comment:
Small nit, but we don't need to repeat the type twice here:

`StringRef DebugLineStrRef(DebugLineData->data(), DebugLineData->size());`

Reply:
Yep! Good catch!